.github/scripts/plugin-helper-gh-pr-create.sh

```diff
@@ -40,8 +40,8 @@ vault_pr_url=$(echo "$resp" | jq '.html_url')
 echo "Vault PR number: $vault_pr_url"
 
 # add labels to Vault PR
-#gh pr edit "$vault_pr_num" --add-label "dependencies,pr/no-changelong,pr/no-milestone" --repo hashicorp/vault
```

Comment:
This change requires us to use a different token than the one we originally grabbed, because that one doesn't appear to have permissions for labelling